packages/grafana-schema/src/common/table.cue

```diff
@@ -11,12 +11,21 @@ TableCellDisplayMode: "auto" | "color-text" | "color-background" | "color-backgr
 // or a gradient.
 TableCellBackgroundDisplayMode: "basic" | "gradient" @cuetsy(kind="enum",memberNames="Basic|Gradient")
 
-// TODO docs
+// Sort by field state
 TableSortByFieldState: {
 	displayName: string
 	desc?:       bool
 } @cuetsy(kind="interface")
 
+// Footer options
+TableFooterOptions: {
+	show: bool
```

Comment:
Would it be possible to document each of these fields? (same for `TableSortByFieldState` and other types). The general rule is that each type property should have a comment describing what it does (even if it's obvious from the name). This will be generated as docs and we don't want them to have empty descriptions, especially considering they will be available for external use.